Assets/Shopify/Examples/Scripts/Panels/CartPanel.cs

```diff
@@ -44,7 +44,8 @@ public class CartPanel : MonoBehaviour {
         private void Start() {
             BackToProductsButton.onClick.AddListener(() => OnReturnToProducts.Invoke());
             CheckoutButton.onClick.AddListener(() => {
-                _cart.CheckoutWithWebView(
+				_cart.CheckoutWithNativePay(
```

Comment:
I would keep this out of this patch for now until we add native pay to the panel.

Reply:
https://github.com/Shopify/unity-buy-sdk/pull/346/commits/e8984029244952e3107f7db6b5d35bcc65dd5906